README.md

```diff
@@ -1,19 +1,97 @@
 [![JetBrains team project](http://jb.gg/badges/team.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 [![GitHub license](https://img.shields.io/badge/license-Apache%20License%202.0-blue.svg?style=flat)](https://www.apache.org/licenses/LICENSE-2.0)
 
-# Debugger memory agent
-This is a native [JVMTI](https://docs.oracle.com/javase/8/docs/platform/jvmti/jvmti.html) agent for JVM, which is currently used in the IDEA debugger to perform basic heap diagnostic during the debug process. To attach the agent run the JVM 
-with the parameter: 
-
-`-agentpath:/path/to/agent`
-
-# Available features
-After attaching this agent to JVM it can:
-1. Calculate shallow and retained sizes by classes;
-2. Calculate shallow and retained sizes by objects;
-3. Find paths to closest gc roots from an object;
-4. Calculate shallow and retained size of an object and get its retained set.
-5. Find reachable objects of class in heap with handling weak/soft references.
+# Memory agent
+This is a native [JVMTI](https://docs.oracle.com/javase/8/docs/platform/jvmti/jvmti.html) agent for a JVM, which is currently used in the IDEA debugger to perform basic heap diagnostic during the debug process. It can also be used in any project, that is running on a JVM, to attach allocation sampling listeners for [low-overhead heap profiling](https://openjdk.java.net/jeps/331) and to collect information about the heap, such as retained sizes, object reachability, and paths to closest GC roots.
```

Comment:
Worth adding a link to Intellij documentation for IntelliJ Debugger where the agent is used